packages/view/backend/sirius-components-view-diagram/src/main/resources/model/diagram.ecore

```diff
@@ -131,6 +131,7 @@
         eType="ecore:EDataType ../../../../../sirius-components-view/src/main/resources/model/view.ecore#//InterpretedExpression"
         defaultValueLiteral="1"/>
     <eStructuralFeatures xsi:type="ecore:EAttribute" name="showIcon" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EBoolean"/>
+    <eStructuralFeatures xsi:type="ecore:EAttribute" name="labelIcon" eType="ecore:EDataType http://www.eclipse.org/emf/2002/Ecore#//EString"/>
```

Comment:
I think `labelIcon` should rather be in `view.ecore#LabelStyle`. What do you think about it? Am I missing something here?

Reply:
Yes, I think you're right, I'm moving it

Reply:
Moving it will imply that custom icons can be used with forms so it will not be moved in this PR.